### ISSUE TITLE
No way to focus "Load All" option in the Picker control when suggestions are available

### ISSUE THREAD

**Issue by HarkeshS** [OfficeDev/office-ui-fabric-react#10757]
### Environment Information

OS : Windows 10 
Browser: Edge
business-app/fabric: 3.9.1
### Describe the issue:

Focus is not coming on "Load All" option in the Picker control when tab is pressed.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/poonam0025/pen/ZEEEZmL 

![PickerControl](https://user-images.githubusercontent.com/53938674/66469907-5d29c500-eaa6-11e9-8551-c802b19e7ff7.png)

#### Actual behavior:

Focus is not coming on "Load All" option in the Picker control when tab is pressed.

#### Expected behavior:

Focus should come on "Load All" option in the Picker control when tab is pressed.

**Comment by ecraig12345** [comment]
Thanks for reporting this! To clarify, the repro steps are:
1. Focus on the picker
2. Start typing letters **which will match an option**--for this example, type **r** (it works properly when there are no matching options)
3. Attempt to reach the "Load all" button. Arrowing down into the menu never goes to "Load all" and pressing tab auto-completes the option.

@joschect Do you have any ideas about the proper behavior here?

**Comment by ecraig12345** [comment]
Reiterating the current behavior for how to access "Load all" with the keyboard:
- When there are *no* suggestions, press tab (from inside the text field)
- When there *are* suggestions...no way to get there (arrowing skips back to the first suggestion, and pressing tab completes a suggestion)

The solution that makes the most sense to me here is to make "Load all" focusable **only** using arrow keys (regardless of whether suggestions are available), since it otherwise looks/acts like part of the menu. @amyngu and/or @joschect does that make sense to you?

**Comment by joschect** [comment]
@ecraig12345 It looks like there is a bug somewhere in suggestions. It should be selecting the "search for more" when you press arrow down but for some reason it isn't. I think having load all be focusable when you tab if there is nothing selected is alright behavior for now. The bigger problem is that you can't reach it by arrowing.
